_posts/2019-10-24-municipal-boundaries-update-for-third-quarter-2019-calendar-year.md

```diff
@@ -0,0 +1,51 @@
+---
+title: 'Municipal Boundaries update for Third Quarter 2019 (calendar year)'
+author:
+  display_name: Mike Heagin
+  email: mheagin@utah.gov
+date: 2019-10-24 21:39:38
+categories:
+  - SGID Blog
+tags:
+  - city
+  - town
+  - municipalities
+  - boundaries
+  - annexations
+---
+
+It’s already over three-quarters of the way through the year, things have slowed down while everyone is enjoying the summer, yet we’re still trying to catch up here. We still expect annexations to pick up some as we head for the end of the year, plenty of procrastinators out there. Here is our updates for the third quarter.
+
+As always these changes to town and city boundaries can affect other departments too, be sure to share your data with others.
+
+We still want everyone to remember the 2020 census is coming up we are still pushing to get everything as perfect as we can so take a look at the state’s data and see if we are missing anything or if there is something strange, and let us know.
```

Comment:
```suggestion
First, we want to remind everyone that the 2020 census is coming up, and we are still pushing to get the State's data layers as perfect as we can. So please take a look at the State’s data and see whether we are missing anything, or if there is something strange, let us know.
```

Reply:
Maybe we should add a "Reminder" or "Census 2020" heading? Also, maybe we should move the census info before the reminder about changes to town/city boundaries affecting other data layers? I feel like that should go closer to the actual updates.